node/src/command.rs

```diff
@@ -58,23 +58,23 @@ fn load_spec(id: &str) -> std::result::Result<Box<dyn sc_service::ChainSpec>, St
 	Ok(match id {
 		"dev" => Box::new(chain_spec::development_config(2000.into())),
 		"template-rococo" => Box::new(chain_spec::local_testnet_config(2000.into())),
-		"egg-rococo" => Box::new(chain_spec::rococo::egg_rococo_config(2003.into())),
+		"egg-rococo" => Box::new(chain_spec::rococo::egg_rococo_config(4006.into())), // Rococo para-id 4006
```

Comment:
Please update the README.md with updated parachain ID in the chainspec section.

Reply:
https://github.com/webb-tools/egg-net/pull/33/commits/f91388f2f2349dd010cccc8cff58f884ade3cefc